shared/domains/chelonia/internals.js

```diff
@@ -1297,6 +1303,9 @@ export default (sbp('sbp/selectors/register', {
   },
   'chelonia/private/in/syncContract': async function (contractID: string, params?: { force?: boolean, resync?: boolean }) {
     const state = sbp(this.config.stateSelector)
+    if (state.contracts[contractID] === null) {
+      throw new Error('Cannot sync permanently deleted contract ' + contractID)
+    }
```

Comment:
Won't this affect other code that relied on the previous `410` error messages?

Reply:
I'd think they're on different codepaths, but sure, I can prefix the error with `410: `

Reply:
(The reason they're different codepaths is that the `410:` is what triggers the permanent removal0